index.ts

```diff
@@ -101,7 +117,7 @@ async function main() {
         } else {
           // issue does not exist in devpool
           // if issue is "closed" then skip it, no need to copy/paste already "closed" issues
-          if (projectIssue.state === "closed") continue;
+          if (projectIssue.state === "closed" || projectIssue.assignee?.login) continue;
```

Comment:
@BeanieMen Current issue is about deleting tweets. Why did you make this change?

Reply:
I just noticed even Alex is laughing at me 😭

Reply:
Also I fixed the dumbass mistake